.github/workflows/deploy-orchard1-to-azure-app-service.yml

```diff
@@ -0,0 +1,201 @@
+name: Deploy Orchard 1 to Azure App Service
+
+concurrency:
+  group: AzureDeployApp
+
+on:
+  workflow_call:
+    secrets:
+      CHECKOUT_TOKEN:
+        required: false
+        description: >
+          The GitHub token to authenticate checkout. Pass in a GitHub personal access token if authenticated submodules
+          are used.
+      AZURE_APP_SERVICE_DEPLOYMENT_SERVICE_PRINCIPAL:
+        required: true
+      AZURE_APP_SERVICE_PUBLISH_PROFILE:
+        required: true
+      MAINTENANCE_USER_NAME:
+        required: false
+      MAINTENANCE_PASSWORD:
+        required: false
+
+    inputs:
+      cancel-workflow-on-failure:
+        description: When set to "true", will cancel the current workflow run with all jobs if this workflow fails.
+        required: false
+        type: string
+        default: "false"
+      timeout-minutes:
+        required: false
+        type: number
+        default: 360
+        description: Configuration for the timeout-minutes parameter of the workflow. The 360 is GitHub's default.
+      machine-type:
+        required: false
+        type: string
+        default: windows-2022
+        description: The machine type to run the workflow under, e.g. "windows-2022". Needs to be Windows.
```

Comment:
Why does it need to be Windows? Same in [msbuild-and-test.yml](https://github.com/Lombiq/GitHub-Actions/pull/221/files#diff-54ef429e49e3be61d02e535386b9946d6a494e900038b7d7a41c30f098f08b03)

Reply:
These are designed for Orchard 1 applications, so we need to use MSBuild, which is Windows-only.